apps/web/src/hooks/useAgentDescriptions.ts

```diff
@@ -0,0 +1,59 @@
+import { scan } from 'promptl-ai'
+import { DocumentVersion } from '@latitude-data/core/schema/types'
+import { resolveRelativePath } from '@latitude-data/constants'
+import { useState, useEffect } from 'react'
+
+export function useAgentDescriptions({
+  documentVersions,
+  selectedAgents,
+  currentDocument,
+}: {
+  documentVersions?: DocumentVersion[]
+  selectedAgents: string[]
```

Comment:
Name this more generically. It can be used for any document. `documentPaths` for example